app/controllers/admin/agents_controller.rb

```diff
@@ -2,10 +2,11 @@
 
 class Admin::AgentsController < AgentAuthController
   respond_to :html, :json
+  before_action :set_scoped_organisations
```

Comment:
Est-ce que cette action est utile en dehors de l'index ?
J'imagine que nous pourrions préciser 
```suggestion
  before_action :set_scoped_organisations, only: :index
```

Voir placer l'appel à cette méthode dans la méthode index directement.